src/storage/txn/commands.rs

```diff
@@ -240,10 +240,15 @@ impl From<MvccGetByStartTsRequest> for TypedCommand<Option<(Key, MvccInfo)>> {
     }
 }
 
+pub trait CommandExt {
+    fn tag(&self) -> metrics::CommandKind;
+    fn incr_cmd_metric(&self);
+}
+
 macro_rules! command {
     (
         $(#[$outer_doc: meta])*
-        $cmd: ident -> $cmd_ty: ty {
+        $cmd: ident, $cmd_ty: ty, $tag_and_cmd_metric: ident {
```

Comment:
Why change `->` to `,`?

Reply:
Also I think just using `tag` rather than `tag_and_cmd_metric` is clearer

Reply:
> Why change `->` to `,`?

I think `Prewrite -> Vec<Result<()>>, prewrite` syntax is a little bit ambiguous when reading. 
Is it 
`(Prewrite -> Vec<Result<()>>), prewrite`
or 
`Prewrite -> (Vec<Result<()>>, prewrite)`
?

But I'm not so sure about what's the proper syntax here, especially there would be more things to put here (eg. `ts`) in the future, what do you have in mind?

Reply:
Ah, it is a good point! I think just using commas is fine. There is no proper syntax here, just whatever is easiest to read.